.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -52,7 +43,7 @@ jobs:
       - name: install Go
         uses: actions/setup-go@v5
         with:
-          go-version: ${{ matrix.go-version }}
+          go-version-file: go.mod
```

Comment:
It will use the latest patch release, like:
- For release-1.9 [uses go1.20.14](https://github.com/karmada-io/karmada/actions/runs/8949460510/job/24583790290?pr=4897#step:4:9)
- For release-1.8 [uses go1.20.14](https://github.com/karmada-io/karmada/actions/runs/8949460510/job/24583790195?pr=4897#step:4:9)
- For release-1.7 [uses go1.20.14](https://github.com/karmada-io/karmada/actions/runs/8949460510/job/24583790104?pr=4897#step:4:9)

It is acceptable I think.

Reply:
+1